SCF-Session-7-Assignment/src/question1/SparseMatrix.java

```diff
@@ -0,0 +1,258 @@
+package question1;
+
+import java.util.Arrays;
+
+
+public class SparseMatrix {
```

Comment:
Make this class as Immutable and follow all the properties of Immutable class.

Reply:
Made the class final